.github/workflows/golangci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          ref: main
+          ref: ${{ github.head_ref }}
```

Comment:
I think you can omit this, as this is the default value (it uses the branch ref that triggered the workflow)

Reply:
Do you mean to do what https://github.com/edgelaboratories/daycount/pull/65/commits/e454437163f29d2e7fd675489166f8b6f285b976 was doing? It did not work